modules/backend/widgets/Form.php

```diff
@@ -160,6 +160,7 @@ public function bindToController()
     protected function loadAssets()
     {
         $this->addJs('js/october.form.js', 'core');
+        $this->addJs('/modules/backend/assets/vendor/mask/jquery.mask.min.js', 'core');
```

Comment:
Does this support subfolder installations?

Reply:
@gergo85 did you test on a install of October where it lives in /cms? I.e. `example.com/cms/modules...`

Reply:
@LukeTowers I installed a new October into subfolder. I tested this feature and it works perfectly. Could I send an account for demo webpage?